packages/console/src/hooks/use-tenant-pathname.ts

```diff
@@ -65,8 +65,20 @@ function useTenantPathname(): TenantPathname {
   const href = useHref('/');
 
   const match = useCallback(
-    (pathname: string, exact = false) =>
-      matchPath(joinPath(':tenantId', pathname, exact ? '' : '*'), location.pathname) !== null,
+    (pathname: string, exact = false) => {
+      // Match relative pathnames directly
+      if (pathname.startsWith('.')) {
+        return (
+          matchPath(joinPath(location.pathname, pathname, exact ? '' : '*'), location.pathname) !==
```

Comment:
Does `joinPath` handle the `.` internally, or do we have to remove the `.` before joining them?
Also, does it also cover the `../` case?

Reply:
it does